FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/AppProperties.kt

```diff
@@ -175,10 +176,8 @@ fun AppBarMenu() {
                 }
             }
 
-            ListItem.SectionHeader(
-                title = stringResource(id = R.string.choose_brand_theme),
-                enableChevron = false,
-                style = SectionHeaderStyle.Subtle,
+            ListItem.Item(
```

Comment:
Lets not make it an item. Keep it as section description and lets not make it clickable. May be a change in the SectionHeader API

Reply:
Following the same API behaviour we have implemented for items:

If onClick is not passed or ( = null) is passed, sectionHeader won't be clickable.

Reply:
For further reviews please consider this new PR:
https://github.com/microsoft/fluentui-android/pull/502